src/repository/aspa.rs

```diff
@@ -184,7 +186,9 @@ impl ProviderAsSet {
                         let current_as_id = provider_as.provider();
                         if let Some(last_as_id) = last {
                             if last_as_id >= current_as_id {
-                                return Err(decode::Malformed.into());
+                                return Err(cons.content_err(
+                                    "provider AS set not in order"
```

Comment:
s/not in order/out of order/ ?

Reply:
‘Provider AS set out of order’ sounds misleading to me, sort of saying that the set isn’t where it should be.

I rewrote this to ‘provider AS set is not ordered’